nipype/interfaces/afni/utils.py

```diff
@@ -1230,6 +1230,116 @@ class To3D(AFNICommand):
     output_spec = AFNICommandOutputSpec
 
 
+class UnifizeInputSpec(AFNICommandInputSpec):
+    in_file = File(
+        desc='input file to 3dUnifize',
+        argstr='-input %s',
+        position=-1,
+        mandatory=True,
+        exists=True,
+        copyfile=False)
+    out_file = File(
+        name_template='%s_unif',
+        desc='output image file name',
+        argstr='-prefix %s',
+        name_source='in_file')
```

Comment:
actually `out_file` is really a file, this is what AFNI expects even if it is called prefix.
I can not change the name to `prefix`, because `AFNICommandInputSpec` expects an input named `out_file` and adds it in any case.